### DISCUSSION TITLE
How to format conversational text for Text Intelligence features

### DISCUSSION THREAD

**Discussion by sgodin** [deepgram/community#919] [General help]
Hello Deepgram devs...

I am using the streaming API's to do real-time Speech to Text for Transcriptions.  Since with the real-time STT service, I cannot layer in Summary and Sentiment Analysis features (ie: Audio Intelligence), I need to use the Text Intelligence feature once the conversation is over.  https://developers.deepgram.com/docs/text-intelligence   

I have multiple channels in the source audio and each channel is associated to a speaker (not using diartization, but that's irrelevant).  **I want to format the text I send to the Text Intelligence API's so that the engine is aware of what each speaker is saying.**  I've had some success  by pre-fixing each utterance with a "Speaker X:" tag.  But I can't find any documentation for this.

What is the official guidance on this?  Can X be a name instead of a channel number?  

For example:
Speaker 0: {some text}
Speaker 1: {some text}
Speaker 1: {more text}
etc.

Is this formatting recommended or something else?  

Also, can the text include a timestamp without effecting the intelligence engine? 

00:05 Speaker 0: {some text}
00:10 Speaker 1: {some text}
00:12 Speaker 1: {more text}

Is there a way to put text (like a timestamp) into some special formatting/bracketing, so it will get ignored by the intelligence engine?

Thanks for your time!

Scott

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by deepgram-community** [comment]
hey sgodin 👋🏻

I believe you can simply provide it markup like you're suggesting.

```txt
speaker 1:
<text>
speaker 2:
<text>
```
Suggest rather than coding this, try it out on our playground instead: [playground intelligence](https://playground.deepgram.com/?endpoint=read&amp;summarize=v2&amp;topics=true&amp;intents=true&amp;sentiment=true&amp;language=en|https://playground.deepgram.com/?endpoint=read&amp;summarize=v2&amp;topics=true&amp;intents=true&amp;sentiment=true&amp;language=en)

> luke

**Reply by sgodin** [reply]
I have been playing around in the playground, but getting inconsistent results.  For example sometimes it seems to pick up my speaker separation tags and return a good summary, and other times it uses the timestamps and tags in the summary text itself.

Are there no hard and fast rules for doing this?  Can you help me understand how the engine works in this manner?
